### PR TITLE
Stack and Dual Monitor Changes [Rebase & FF]

### PR DESCRIPTION
**Pull request by makubacki**
## Description

**Changes are accumulating here for in progress work**

---

- [x] Impacts functionality?
- [ ] Impacts security?
- [ ] Breaking change?
- [ ] Includes tests?
- [ ] Includes documentation?

## How This Was Tested

- Build and boot

## Integration Instructions

- N/A

### CODE REVIEW

**Review by Javagedes — APPROVED** [review]

File: SeaPkg/Core/Init/VmcsInit.c [hunk 1]
```diff
@@ -32,13 +35,15 @@ _ModuleEntryPoint (
...
-  IN UINT64  *Vmcs
+  IN UINT32   Index,
+  IN UINT64   *Vmcs,
+  IN BOOLEAN  IncrementGuestRip
   )
 {
   // VM_EXIT_CONTROLS                             VmExitCtrls;
   // VM_ENTRY_CONTROLS                            VmEntryCtrls;
   // GUEST_INTERRUPTIBILITY_STATE  GuestInterruptibilityState;
+  // UINT64 Data64;
```
Comment (kuqin12) [inline SeaPkg/Core/Init/VmcsInit.c:46]: nit: unnecessary change?

**Review by kuqin12 — APPROVED** [review]

File: SeaPkg/Core/Init/VmcsInit.c (hunk 1 above)
Comment (makubacki) [reply SeaPkg/Core/Init/VmcsInit.c:46]: Yes, I'll remove it.

Comment (makubacki) [reply SeaPkg/Core/Init/VmcsInit.c]: Done
